models.py

```diff
@@ -593,8 +593,8 @@ def Fetch(cls, institution, session, email):
 class ClassRoster(ndb.Model):
   # comma separated list of student emails
   student_emails = ndb.TextProperty()
```

Comment:
I wonder which is faster, comma separated student_emails like this or changing this to another JsonProperty with an array of student emails. I imagine the comma separated student emails is faster, but I could be totally wrong.